schemas/com.snowplowanalytics.snowplow.enrichments/pii_enrichment_config/jsonschema/1-1-0

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for PII pseudonymization enrichment",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.enrichments",
+    "name": "pii_enrichment_config",
+    "format": "jsonschema",
+    "version": "1-1-0"
+  },
+  "type": "object",
+  "properties": {
+    "enabled": {
+      "type": "boolean"
+    },
+    "emitIdentificationEvent": {
```

Comment:
Shouldn't this be `emitPiiTransformationEvent`?

Reply:
Or actually I prefer `emitEvent`, to make it less coupled to the event name

Reply:
Done